lib/matplotlib/colorbar.py

```diff
@@ -1240,8 +1240,18 @@ def _proportional_y(self):
         if (isinstance(self.norm, colors.BoundaryNorm) or
                 self.boundaries is not None):
             y = (self._boundaries - self._boundaries[self._inside][0])
-            y = y / (self._boundaries[self._inside][-1] -
-                     self._boundaries[self._inside][0])
+
+            # original
```

Comment:
Please do not keep the original code around.  Because the code is in git and there are a number of UIs (inculding this website) for viewing the diffs it is much clearer to simply remove the old code and replace it with the new code.  If we kept the full history of Matplotib in-line like this it would very quickly become impossible to read!